src/main/java/vazkii/botania/common/block/tile/corporea/TileCorporeaIndex.java

```diff
@@ -372,12 +374,25 @@ public static void clearCache() {
 		serverIndexes.clear();
 	}
 
+	private static IFormattableTextComponent formatRequest(int inputCount, ITextComponent requestName, ICorporeaResult res) {
```

Comment:
just put this on the result class?

Reply:
or on the request. The count can be saved from the initial count, the name can be gotten from the matcher.

Reply:
Problem is that the `CorporeaRequest` object never gets handled by TileCorporeaIndex, it's all inside CorporeaHelper.requestItem()